code/game/atoms.dm

```diff
@@ -86,6 +86,9 @@
 		return 0
 	return -1
 
+/atom/proc/grants_equipment_vision(var/mob/user)
```

Comment:
This should have a dmdoc that explains what is the purpose of this function, and what it returns